docs/migrations/v2-to-v3.md

```diff
@@ -42,11 +42,38 @@ app.UpgradeKeeper.SetUpgradeHandler("v3",
             ControllerEnabled: true, 
         }
 
-        // create ICS27 Host submodule params
+        // create ICS27 Host submodule params 
         hostParams := icahosttypes.Params{
             HostEnabled: true, 
-            AllowMessages: []string{"/cosmos.bank.v1beta1.MsgSend", ...}, 
-        }
+			hostParams := icahosttypes.Params{
+					HostEnabled: true,
+                    // here you can add any transaction supported by your chain, and ica controllers will be able to make
+                    // those transactions on your chain.
+					AllowMessages: []string{
```

Comment:
I agree that adding an extra explanation could be helpful. Maybe also adding the link to the section of the docs that explains the [`AllowMessages` parameter](https://github.com/cosmos/ibc-go/blob/main/docs/apps/interchain-accounts/parameters.md#allowmessages).
But I don't think we should add a long list of SDK messages here; with one or two as example, I think it's more than enough.

Reply:
Thanks for taking the time to add all these messages. I appreciate the improvement to the migration docs!

I do agree that explicitly providing an example will create a sort of "default" within the ecosystem, but I think it's important to respect the purpose of the `AllowMessages` parameter. The intent of adding such a parameter was to limit the functionality a controller chain could perform on the host chain since the messages executed by interchain account transactions does not follow the exact same authentication path as regular transactions (although in theory they should be equivalent). Thus, in my opinion, providing few examples and explaining that any message could be added is perhaps the best way to get each chain to actually consider the implications here. We shouldn't be providing a default. The default allowed messages should be empty (which is why this needs to be set during an upgrade handler)